package.json

```diff
@@ -400,9 +400,100 @@
 							"verticalOffset": 0
 						}
 					}
+				},
+				"cursorless.wrapperSnippetPreferences": {
+					"description": "Allows snippets to signal what their default scope types etcetera should be",
```

Comment:
Down the road we may allow signalling things like `selectionType`, etc, so I figured let's have this be a map